app/scripts/controllers/preferences.js

```diff
@@ -94,6 +94,7 @@ export default class PreferencesController {
         featureNotificationsEnabled: false,
         showTokenAutodetectModal: null,
         showNftAutodetectModal: null, // null because we want to show the modal only the first time
+        isConfirmationAdvancedDetailsOpen: false,
```

Comment:
`isConfirmationAdvancedDetailsOpen` sounds more like app state then preference

Reply:
It is an app state though, we substituted `useState` with this background preference property for expanding and collapsing advanced details...

Reply:
Even though it's triggered via a toggle, it is still a mechanism to indicate their preference which we could theoretically use elsewhere.

So maybe `showConfirmationAdvancedDetails`?